test/sql-tap/in2.test.lua

```diff
@@ -29,7 +29,7 @@ test:do_execsql_test(
         -- </in2-1>
     })
 
-local N = 2000
+local N = 1000
```

Comment:
I think it would be about the same as removing the test or moving it to long/perf tests. I suggest leaving everything as is, at least we will know that this test is not actually for Tarantool.

Reply:
I guess that N = 1000 doesn't reproduce the original bug in sqlite (otherwise it would be used from scratch), so the comments regarding sqlite's bug would be misleading in the case.

If we still link the original bug, we should mention somehow that the test case has been changed in a way that it may not be a reproducer for the bug anymore.

Anyway, it is not a blocker, just note. I'll close this discussion.